src/binding.cc

```diff
@@ -3,18 +3,34 @@
 #include <stdlib.h>
 #include <string.h>
 #include <string>
+#include <thread>
 
 #include "third_party/v8/include/libplatform/libplatform.h"
 #include "third_party/v8/include/v8.h"
 #include "third_party/v8/src/base/logging.h"
 
-#include "deno.h"
-#include "internal.h"
+#include "src/deno.h"
+#include "src/internal.h"
 
 namespace deno {
 
 static bool skip_onerror = false;
 
+void InitializeCommon(Deno* d, void* data, deno_recv_cb recv_cb,
+                      deno_cmd_id_cb cmd_id_cb) {
+  d->current_args = nullptr;
+  d->current_cmd = nullptr;
+  d->recv_cb = recv_cb;
+  d->cmd_id_cb = cmd_id_cb;
+  d->data = data;
+
+  auto env_value = getenv("DENO_THREADS");
+  d->threads_enabled = env_value != nullptr && env_value == std::string("1");
```

Comment:
Since d is opaque to rust, `threads_enabled` should be an argument to `deno_new`... or maybe something like `deno_enable_theads(bool)`...